### DISCUSSION TITLE
Arrange Badges Alphabetically

### DISCUSSION THREAD

**Discussion by Ileriayo** [Ileriayo/markdown-badges#166] [Ideas]
You can always use `Ctrl + F` or `Cmd + F` to search for specific badges, but it would also fun to implement a solution to have the [badges sorted alphabetically](https://github.com/Ileriayo/markdown-badges/issues/154) #154.

I have thought about solving this by having a pre-commit hook that lints the code and fixes the arrangement where necessary. What do you think, how would you achieve this? 🤓

**Comment by ValentinNeaga** [comment]
Hello @Ileriayo,

I've created a pull request (#173) to order everything alphabetically since I'm not experienced with pre-commit hooks.

**Reply by Ileriayo** [reply]
This was a temporary solution, but was worth it!

Thanks @JakyeRU!

**Reply by ValentinNeaga** [reply]
It is my pleasure, @Ileriayo. You're right, this is a temporary solution. Currently, I'm experimenting with pre-commit hooks. Maybe I'll be able to automate this process in the future.

**Reply by Ileriayo** [reply]
Looking forward to that!

**Comment by mehtaAnsh** [comment]
Hello guys,

The PR has been merged successfully! We would love to have a solution though for pre-commit hooks. I would love to know about it as well :)

**Comment by Ileriayo** [comment]
This was solved again (manually) by PR #292.
We are yet to have an automated solution, however.

**Comment by Jukelyn** [comment]
Just wondering, is this going to be addressed at some point?

**Reply by Ileriayo** [reply]
Open to contributions.

**Comment by Siddhi-Iccha** [comment]
Thank you @Ileriayo for pointing me to this discussion.

Linking [issue-#892](https://github.com/Ileriayo/markdown-badges/issues/892) here so that others can also provide their inputs
